src/App/Tracker/Controller/Hooks/ReceiveCommentsHook.php

```diff
@@ -128,7 +128,22 @@ protected function insertComment()
 			$this->pullUserAvatar($this->hookData->comment->user->login);
 		}
 
-		// $this->triggerEvent('onCommentAfterCreate', $table);
+		try
+		{
+			// Get a table object for the new record to process in the event listeners
+			$issuetable = new IssuesTable($this->db);
+			$issuetable->load(array('issue_number' => $this->hookData->issue->number));
```

Comment:
1. This will not work as `issue_number` is not unique. You need to add the project ID here also:
```php
$issuetable->load(
	array(
		'issue_number' => $this->hookData->issue->number,
		'project_id' => $this->project->project_id
	)
);
```

That will be a unique combination (issue_number + project_id key is defined as unique for `#__issues` table).

2. Please use camelCase for vars: `$issuetable` -> `$issueTable`